pymc/distributions/continuous.py

```diff
@@ -344,7 +344,7 @@ class FlatRV(RandomVariable):
     _print_name = ("Flat", "\\operatorname{Flat}")
 
     @classmethod
-    def rng_fn(cls, rng, size):
+    def rng_fn(cls, rng, *args):
```

Comment:
I thought size might be passed as kwarg by Aesara, but then it would have failed (assuming we have coverage). In any case it can still exist in RandomVariables and rng_fn because those are always defined in terms of size

Reply:
The superclass doesn't have it in the signatures and also many other distributions don't have it.

Generally we shouldn't override signatures - it angers the typechecking gods.

Reply:
I strongly disagree with this change. It's the `rng_fn` that aesara needs and we should use aesara's expected signature for it. I think that we should leave `size` here since a `RandomVariable` `Op` is low level enough as to not confuse beginners.

Reply:
That's fine with me, ist just think we should be consistent.
I'll revert the changes in `rng_fn` signatures.
```suggestion
    def rng_fn(cls, rng, size):
```